diff --git a/src/libbson/CMakeLists.txt b/src/libbson/CMakeLists.txt
index 7c479d8b5..411015140 100644
--- a/src/libbson/CMakeLists.txt
+++ b/src/libbson/CMakeLists.txt
@@ -318,7 +318,7 @@ if (ENABLE_EXAMPLES)
 endif () # ENABLE_EXAMPLES
 
 set (BSON_HEADER_INSTALL_DIR
-   "${CMAKE_INSTALL_INCLUDEDIR}/libbson-${BSON_API_VERSION}"
+   "${CMAKE_INSTALL_INCLUDEDIR}"
 )
 
 if (ENABLE_STATIC MATCHES "ON|AUTO")
diff --git a/src/libmongoc/CMakeLists.txt b/src/libmongoc/CMakeLists.txt
index e744f0e92..2fb275916 100644
--- a/src/libmongoc/CMakeLists.txt
+++ b/src/libmongoc/CMakeLists.txt
@@ -979,7 +979,7 @@ else ()
 endif ()
 
 set (MONGOC_HEADER_INSTALL_DIR
-   "${CMAKE_INSTALL_INCLUDEDIR}/libmongoc-${MONGOC_API_VERSION}"
+   "${CMAKE_INSTALL_INCLUDEDIR}"
 )
 
 install (
